cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/AgentIntervalProvider.java

```diff
@@ -24,10 +24,16 @@
 public interface AgentIntervalProvider {
     public static class Interval {
         final long interval;
+        final long errorInterval;
         final long timeout;
 
         public Interval(long interval, long timeout) {
+            this(interval, interval, timeout);
+        }
+
+        public Interval(long interval, long errorInterval, long timeout) {
```

Comment:
Think it makes sense to group the intervals together. It's a whole new constructor so there aren't any limitations on what can be changed.